src/wp-admin/includes/dashboard.php

```diff
@@ -1470,7 +1470,8 @@ function cp_dashboard_upgrade() {
 		echo '<p>' . sprintf( __( 'You can continue to use ClassicPress %s at this time. You will be notified when a new version is available that supports future versions of PHP. <a href="%s">Learn more</a> about testing version 2.0.0.' ), $display_version, esc_url( 'https://www.classicpress.net/help-test-classicpress-v2/' ) ) . '</p>';
 	}
 
-	echo '<p>' . __( 'Deprecation notice: Version 1.6 is the last version to support PHP 5.6 - 7.3. The minimum required version for 2.0.0 will be PHP 7.4.' ) . '</p>';
+	echo '<p>' . __( '<strong>Deprecation notice:</strong> Version 1.6 is the last version to support PHP 5.6 - 7.3. The minimum required version for 2.0.0 will be PHP 7.4.' ) . '</p>';
+	echo '<p>' . __( '<strong>Deprecation notice:</strong> Version 1.7 is the last version to support XHTML. HTML5 will be the default doctype. Make sure your theme does not use XHTML. Themes will no longer need to declare HTML5 support using <code>add_theme_support( \'html5\', array() )</code> function.' ) . '</p>';
```

Comment:
Just to be more clear for people whose first language is not English, I suggest:

```
echo '<p>' . __( '<strong>Deprecation notice:</strong> Version 1.7 is the last version to support XHTML. ClassicPress Versions after 1.7, will have HTML5 as the default doctype. Themes will <strong>no longer need</strong> to declare HTML5 support using <code>add_theme_support( \'html5\', array() )' ) . </code> function. Ensure your theme does not use XHTML before upgrading.'</p>'
```

Reply:
1.7 should be the last version in 1.x branch. It was supposed to be 1.6, but since we decided to remove XHTML we have to include deprecation notice to comply with SemVer. Version 2.0 will not include deprecation notices. Do you still think we need to check CP version for this?